CMakeLists.txt

```diff
@@ -2,15 +2,22 @@ cmake_minimum_required(VERSION 3.5)
 project(depthimage_to_laserscan)
 include(GenerateExportHeader)
 
-# Default to C++14
+# Default to C++17
 if(NOT CMAKE_CXX_STANDARD)
-  set(CMAKE_CXX_STANDARD 14)
+  set(CMAKE_CXX_STANDARD 17)
 endif()
 
 if(CMAKE_COMPILER_IS_GNUCXX OR CMAKE_CXX_COMPILER_ID MATCHES "Clang")
   add_compile_options(-Wall -Wextra -Wpedantic)
 endif()
 
+message(STATUS "CHRIS: ROS_DISTRO: $ENV{ROS_DISTRO}")
```

Comment:
Debug statement left in ;-)

Reply:
Excellent point.  I don't know how many times I've looked at this in the last 2 weeks, and I never saw it :).  Removed in 3eeba6c